CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
+
+## Next (Unreleased)
+
+*
```

Comment:
When we merge a PR with a notable new feature or bug fix. Let's add a line here.

Then, when we are ready to release, we can add the version number and date and create a new empty "Next" section.